src/main/resources/messages_en.properties

```diff
@@ -557,6 +557,9 @@ invalid.environment.data.id=Invalid environment data Id.
 invalid.variable.for.environment.data=The specified variable does not match for saved environment data.
 study.must.not.have.observation=The study must not have observation.
 methoddbid.invalid=Breeding Method DbId does not exist
+study.delete.not.permitted=You are not able to delete this study as you are not the owner. The owner is {0}.
+study.template.delete.not.permitted=Program templates cannot be deleted.
+study.id.not.exists=The study id {0} does not exist.
```

Comment:
Maybe reuse study.not.exist ?

Reply:
The validatión in fieldbook return the id of the study.